unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/ChatWidgetHUD/DefaultChatEntry.cs

```diff
@@ -40,8 +40,11 @@ public class DefaultChatEntry : ChatEntry, IPointerClickHandler, IPointerEnterHa
         public override event Action OnCancelHover;
         public override event Action OnCancelGotoHover;
 
-        public override void Populate(ChatEntryModel chatEntryModel) =>
+        public override void Populate(ChatEntryModel chatEntryModel)
+        {
+            chatEntryModel.bodyText = ReplaceNonSupportedCharacteres(body.font, chatEntryModel.bodyText, '?');
```

Comment:
this operation should be done inside `PopulateTask` method, since all the curating process is there, what do you think?

Reply:
You're right. Done!